tethys_cli/proxyapps_commands.py

```diff
@@ -0,0 +1,219 @@
+                f"    logo_url: {proxy_app.logo_url}\n"
+                f"    tags: {proxy_app.tags}\n"
+                f"    enabled: {proxy_app.enabled}\n"
+                f"    show_in_apps_library: {proxy_app.show_in_apps_library}\n"
+                f"    back_url: {proxy_app.back_url}\n"
+                f"    open_in_new_tab: {proxy_app.open_in_new_tab}\n"
+                f"    display_external_icon: {proxy_app.display_external_icon}\n"
+                f"    order: {proxy_app.order}"
+            )
+        else:
+            print(f"  {proxy_app.name}: {proxy_app.endpoint}")
+
+
+def update_proxyapp(args):
+    setup_django()
+    from tethys_apps.models import ProxyApp
+
+    app_name = args.name
+    for key, value in args.set_kwargs:
+        app_key = key
+        app_value = value
+
+        try:
+            proxy_app = ProxyApp.objects.get(name=app_name)
+            if not hasattr(proxy_app, app_key):
+                write_warning(f"Attribute {app_key} does not exist")
+                continue
+
+            setattr(proxy_app, app_key, app_value)
+            proxy_app.save()
+
+            write_info(f"Attribute {app_key} was updated successfully with {app_value}")
+
+        except ProxyApp.DoesNotExist:
+            write_error(f"Proxy app named '{app_name}' does not exist")
+            exit(1)
+
+    write_success(f"Proxy app '{app_name}' was updated successfully")
+    exit(0)
+
+
+def add_proxyapp(args):
+    """
+    Add Proxy app
+    """
+    setup_django()
+
+    from tethys_apps.models import ProxyApp
+
+    app_name = args.name
+    app_endpoint = args.endpoint
+    app_description = args.description
+    app_logo_url = args.logo_url
+    app_tags = args.tags
+    app_enabled = args.enabled
+    app_show_in_app_library = args.show_in_apps_library
+    app_back_url = args.back_url
+    app_open_new_tab = args.open_new_tab
+    app_display_external_icon = args.display_external_icon
+    app_order = args.order
+    try:
+        proxy_app = ProxyApp.objects.get(name=app_name)
+        write_error(f"There is already a proxy app with that name: {app_name}")
+        exit(1)
+
+    except ProxyApp.DoesNotExist:
+        try:
+            proxy_app = ProxyApp(
+                name=app_name,
+                endpoint=app_endpoint,
+                logo_url=app_logo_url,
+                back_url=app_back_url,
+                description=app_description,
+                tags=app_tags,
+                show_in_apps_library=app_show_in_app_library,
+                enabled=app_enabled,
+                open_in_new_tab=app_open_new_tab,
+                display_external_icon=app_display_external_icon,
+                order=app_order,
+            )
+
+            proxy_app.save()
+            write_success(f"Proxy app {app_name} added")
+            exit(0)
+        except IntegrityError:
+            write_error(f"Not possible to add the proxy app: {app_name}")
```

Comment:
```suggestion
            write_error(f'Not possible to add the proxy app "{app_name}" because one or more values of the wrong type were provided. Run "tethys proxyapp add --help" to see examples for each argument.')
```